Resources/public/js/views/fields/ez-author-editview.js

```diff
@@ -332,6 +332,15 @@ YUI.add('ez-author-editview', function (Y) {
              * @default undefined
              */
             fieldDefinition: {},
+
+            /**
+             * The logged in user
+             *
+             * @attribute user
+             * @type {eZ.User}
+             * @default undefined
+             */
+            user: {},
```

Comment:
since AuthorEditView extends FieldEditView, you don't have to redeclare the attribute.